### PR TITLE
Feature/mw 615 android crashes if hitting end ride

### PR DESCRIPTION
**Pull request by GrayedFox**

### CODE REVIEW

File: FantasmoSDK/app/src/main/java/com/fantasmo/sdk/filters/FMBlurFilter.kt [hunk 1]
```diff
@@ -61,14 +57,9 @@ class FMBlurFilter(
         if (!::rs.isInitialized) {
             rs = RenderScript.create(context)
             histogram = ScriptIntrinsicHistogram.create(rs, Element.U8(rs))
-            convolve = ScriptIntrinsicConvolve3x3.create(rs, Element.U8_4(rs))
+            convolve = ScriptIntrinsicConvolve3x3.create(rs, Element.U8(rs))
```
Comment (GrayedFox) [inline FantasmoSDK/app/src/main/java/com/fantasmo/sdk/filters/FMBlurFilter.kt:60]: ⚡

**Review by GrayedFox — COMMENTED** [review]
LGTM

**Comment by GrayedFox** [comment]
@bastinagual is this safe to merge or do you want to clean it up a bit? Looks fine to me

**Comment by GrayedFox** [comment]
Also please remember to delete branches after merging them 👍🏾

**Comment by bastinagual** [comment]
This is safe to merge, I just added `imageEnhancementInfo` to the localize request as well on this branch

**Pull request merged**